docker-compose-3-nodes.yaml

```diff
@@ -159,10 +159,18 @@ services:
       - validator-1
       - validator-registry-tp-1
     entrypoint: "bash -c \"\
-        source /project/sawtooth-poet/build_sgx_binary.sh \
+        rm -f /etc/sawtooth/keys/validator.* \
+        && rm -f /poet_keys/poet-config.batch \
+        && while [ ! -f /poet_keys/validator-1.priv ]; do sleep 1; done \
+        && while [ ! -f /poet_keys/validator-1.pub ]; do sleep 1; done \
+        && mkdir -p /etc/sawtooth/keys/ \
+        && cp /poet_keys/validator-1.priv /etc/sawtooth/keys/validator.priv \
+        && cp /poet_keys/validator-1.pub /etc/sawtooth/keys/validator.pub \
+        && cat /etc/sawtooth/keys/validator.pub \
+        && source /project/sawtooth-poet/build_sgx_binary.sh \
         && RUST_BACKTRACE=1 cargo run --bin sawtooth-poet -- \
         --config /project/sawtooth-poet/src/core/src/resources/poet_config.toml \
-        -C tcp://validator-1:5050 -vv \""
+        -C tcp://validator-1:5050 -v --genesis /poet_keys/poet-config.batch\""
```

Comment:
Verbosity: There were too many logs of trying to summarize blocks. It became difficult when having multiple nodes.
Genesis flag: We need genesis flag because all PoET engines are currently using same toml file placed in repo, "genesis" parameter is set to false by default in it.  "genesis" flag was introduced to override the behavior of toml config file for gensis node.

Reply:
The docker-compose 3 nodes file is for test nodes. Additional verbosity is expected.

Reply:
I do not think this is a good approach.

Reply:
Ok, please suggest alternatives

Reply:
Restore the flags to default verbosity.

Reply:
-vv